buildbot-conf/cordova-config.json.sample

```diff
@@ -0,0 +1,45 @@
+{
+    "couchdb": {
+        "uri":  "http://localhost:5984",
+        "host": "localhost",
+        "port": 5984
```

Comment:
It is part of future restructuring efforts. Moved to another branch, because it indeed does not belong in this PR.